src/components/app/App.js

```diff
@@ -17,6 +20,8 @@ function App() {
       {/*  todo: здесь нужно использовать Footer по аналогии с Header */}
 
       {/*  todo: здесь нужно использовать UserCard по аналогии с Header, как пропсу user передать usersList[0]*/}
+        <UserCard usersList = {usersList}/>
```

Comment:
Supposed to be that 1 card gets 1 user at a time, so please change the logic so that UserCard would get only `user` props and it would be usersList[0]